mappings/net/minecraft/world/WorldDataVersion.mapping

```diff
@@ -20,10 +20,10 @@ CLASS net/minecraft/unmapped/C_enlieprc net/minecraft/world/WorldDataVersion
 		ARG 1 other
 	METHOD m_xfuxfldc getSeries ()Ljava/lang/String;
 		COMMENT Returns the series of this version.
-		COMMENT
-		COMMENT <p>This is stored in the {@code Series} field within {@code level.dat} files.
-		COMMENT
-		COMMENT <p>Known values include {@value #MAIN_SERIES} for the main series,
+		COMMENT <p>
+		COMMENT This is stored in the {@code Series} field within {@code level.dat} files.
+		COMMENT <p>
+		COMMENT Known values include {@code main} for the main series,
 		COMMENT and {@code ccpreview} for the caves and cliffs experimental snapshots.
```

Comment:
Might be worth adding `deep_dark_preview` for the 1.19 experimental version

Reply:
Added.